cardano-ping/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Revision history for cardano-ping
 
+## 0.2.0.2 -- 2023-06-08
+
+* Support `NodeToNodeV_11` and `NodeToClientV_16`.
```

Comment:
Shouldn't this be NodeToNodeV_12?

Reply:
I think both `V_11` and `V_12`, I'll remember to add this to `node` release.